src/routers/datatypeRouters/descriptions.ts

```diff
@@ -47,9 +47,9 @@ export const descriptions = {
   ontology_terms_children: 'Retrieve all child nodes of an ontology term.<br> Example: ontology_term_id = UBERON_0003661. The limit parameter controls the page size and can not exceed 500.',
   ontology_terms_parents: 'Retrieve all parent nodes of an ontology term.<br> Example: ontology_term_id = UBERON_0014892. The limit parameter controls the page size and can not exceed 500.',
   ontology_terms_transitive_closure: 'Retrieve all paths between two ontology terms (i.e. transitive closure).<br> Example: ontology_term_id_start = UBERON_0003663, ontology_term_id_end = UBERON_0014892',
-  variants_proteins: 'Retrieve allele-specific transcription factor binding events from ADASTRA by querying variants.<br> \
+  variants_proteins: 'Retrieve allele-specific transcription factor binding events from ADASTRA in cell type-specific context, allele-specific transcription factor binding events from GVATdb, and pQTL from UKB by querying variants.<br> \
   Set verbose = true to retrieve full info on the variant-transcription factor pairs, and ontology terms of the cell types.<br> Example: spdi = NC_000020.11:3658947:A:G, hgvs = NC_000020.11:g.3658948A>G, <br>variant_id = 027a180998e6da9822221181225654b628ecfe93fd7a23da92d1e4b9bc8db152 (internal hashed variant ids), rsid = rs6139176.<br> The limit parameter controls the page size and can not exceed 100.',
-  proteins_variants: 'Retrieve allele-specific transcription factor binding events from ADASTRA in cell type-specific context by querying transcription factors.<br> \
+  proteins_variants: 'Retrieve allele-specific transcription factor binding events from ADASTRA in cell type-specific context, allele-specific transcription factor binding events from GVATdb, and pQTL from UKB by querying transcription factors.<br> \
```

Comment:
Could change 'by querying transcription factors' at the end -> 'by querying proteins' to be more general, since we also have pQTLs now.

Reply:
Will update it.